File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/EnemyOceanDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class EnemyOceanDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playEnemyOceanDragonKillAnimation() {
        new AnimationTester().testAnimation(new EnemyOceanDragonKillAnimation());
    }
}

File: test/hud/animations/EnemyOceanDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {EnemyOceanDragonKillAnimation} from '../../../src/hud/animations/EnemyOceanDragonKillAnimation.js';

/**
 * Integration test for enemy ocean dragon kill animation on actual hardware.
 */
describe('EnemyOceanDragonKillAnimation Integration', () => {
    test('playEnemyOceanDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new EnemyOceanDragonKillAnimation());
    }, 30000);
});
